src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1137,4 +1139,12 @@ public static void forceJVMLocaleToUSEnglish() {
         final List<T> sorted = values.stream().sorted().collect(Collectors.toList());
         return sorted.get(sorted.size() / 2);
     }
+
+    public static java.util.Iterator<String> lineIterator(final Path path) {
```

Comment:
Could you throw in some javadoc for this?

Reply:
Added, let me know if you want to add something else!